test/integration/backward_compatible/parallel/security/SecurityTest.js

```diff
@@ -23,8 +23,17 @@ const TestUtil = require('../../../../TestUtil');
 const fs = require('fs');
 const RC = require('../../../RC');
 const {IllegalStateError} = require('../../../../../lib');
-const {SimpleCredentials} = require('./SimpleCredentials');
-const {TokenEncoding} = require('../../../../../lib/security/TokenEncoding');
+
+// Lazy imports are due to back compatibility
+const getTokenEncoding = () => {
+    const {TokenEncoding} = require('../../../../../lib/security/TokenEncoding');
+    return TokenEncoding;
+};
+
+const getSimpleCredentials = () => {
+    const {SimpleCredentials} = require('./SimpleCredentials');
+    return SimpleCredentials;
+};
```

Comment:
This is not necessary, as it is a custom file that sits in the tests directory

Reply:
Fixed at f93de860